src/wallet/KintoWalletFactory.sol

```diff
@@ -53,9 +53,9 @@ contract KintoWalletFactory is Initializable, UUPSUpgradeable, OwnableUpgradeabl
      * @dev Upgrade calling `upgradeTo()`
      */
     function initialize(IKintoID _kintoID) external virtual initializer {
-        __Ownable_init();
+        __Ownable_init(msg.sender);
         __UUPSUpgradeable_init();
-        beacon = new UpgradeableBeacon(address(_implAddress));
+        beacon = new UpgradeableBeacon(address(_implAddress), address(this));
```

Comment:
whats the second param?

Reply:
it's the owner, this is the new constructor:
```
constructor(address implementation_, address initialOwner) Ownable(initialOwner) {
        _setImplementation(implementation_);
    }
```
I'm passing `address(this)` which is the the factory contract, is that okay, right?

Reply:
yes